src/screens/HomeScreen.tsx

```diff
@@ -1,24 +1,17 @@
 import React, { useCallback, useEffect, useState } from 'react';
-import { t } from 'i18next';
-import { useActiveAccount } from '../hooks/useActiveAccount';
 import { useAppConfig } from '../hooks/useAppConfig';
-import { ActivityIndicatorView } from '../components/ActivityIndicatorView';
 import { TilesList } from '../components/tiles/TilesList';
 import { ScreenSurface } from '../components/ScreenSurface';
 import { HomeStackScreenProps } from '../navigators/types';
-import { useJoinCircles } from '../hooks';
 import { refreshNotifier } from '../common/RefreshNotifier';
 import { HeaderLeftRefreshButton } from '../components/HeaderLeftRefreshButton';
 
 export const HomeScreen = (navProps: HomeStackScreenProps<'Home'>) => {
-  const { isLoading: loadingAccount } = useActiveAccount();
-  const { isInitialLoading: loadingAppConfig, data: appConfig } =
-    useAppConfig();
-  const { isInitialLoading: loadingJoinCircles } = useJoinCircles();
```

Comment:
This appears to be gone now. Need to make sure we add this back so users autojoin circles

Reply:
Nice catch!!

Reply:
I will add this back on the 3rd PR so I don't have to rebase each one. I plan on merging these one after another anyway.